### DISCUSSION TITLE
每次重启系统，防火墙会变成丢弃从wan到lan的转发数据。

### DISCUSSION THREAD

**Discussion by ccettu** [istoreos/istoreos#662] [Q&A] [answered]
反馈bug/问题模板，提建议请删除

## 1.关于你要提交的问题
为避免重复issue，请先搜索issue，确认没有类似issue再提交新issue；
注意搜索时**包括已关闭**的issue（删掉搜索框的的“is:open”条件）；

Q：是否用关键词搜索了issue? (使用 "x" 选择)
* [x] 没有类似的issue

### (1) 具体问题

A：网络-防火墙-区域，从WAN区域转发到LAN区域，或任意区域的入站数据改成接受后，在重启路由器或者做其它关于接口的设置后，该按钮会重新自动变成丢弃，这样会导致网心云docker容器魔方停止数据上传，需要手动再调会接受，但下次启动路由器后，仍然会变成丢弃。

### (2) 路由器型号和固件版本

A：J4125物理机安装和PVE下安装的iStoreOS 21.02.3 2023032001版本和上一个版本都有这样的问题。

### (3) 详细日志和/或截图

A：
![Snipaste_2023-03-21_01-08-22](https://user-images.githubusercontent.com/49177353/226419399-f55fd0e8-1dd1-414e-b6bc-928ff4851e47.jpg)

**Comment by jjm2473** [comment, marked as answer]
这不是早就有相同的issue了吗？ #632

为了防止外网攻击，pppoe拨号时自动禁止wan口入站。
可以自己在流量规则配置入站允许，或者在istore安装systool，里面有关掉自动脚本的功能。

**Comment by gs314520** [comment]
打开etc/config目录下的uhttpd文件，找到option rfc1918_filter '1'，把1改成0即可

**Reply by jjm2473** [reply]
跟这个无关，你说的应该是内网使用外网IP无法访问后台，提示“Rejected request from RFC1918 IP to public server address”之类的错误。

**Comment by w9315273** [comment]
太傻了 谁家的大门会一直开着还不上锁 小偷不偷你都是对你的不尊重

**Reply by jjm2473** [reply]
对，有端口开放需求的的配置个规则就行了

**Comment by yangnansc** [comment]
需要下载系统工具，里面有个disable-wandrop关闭就可以了

**Reply by jannson** [reply]
我去，想不到还有人用这个。昨天才把这个脚本删除了。。。

**Comment by yangnansc** [comment]
![Uploading 企业微信截图_20250101195924.png…]()

**Comment by jjm2473** [comment]
现在默认已经关闭wan访问，不需要再关注
